src/cli/commands/test/iac/output.ts

```diff
@@ -183,6 +184,8 @@ export function buildOutput({
         ? new FormattedCustomError(
             errorResults[0].message,
             formatFailuresList(allTestFailures),
+            undefined,
+            new CLI.GeneralIACFailureError(formatFailuresList(allTestFailures)),
```

Comment:
suggestion: will need to remove formatting using `stripAnsi` here:
```suggestion
            new CLI.GeneralIACFailureError(stripAnsi(formatFailuresList(allTestFailures))),
```

Reply:
I assume that FormattedCustomError is a customer error, so we might be able to do the assignment and stripping of formatting more centrally, similar to the CustomError.